app/(landing)/local-elections-webinar/components/LocalElectionsWebinarPage.js

```diff
@@ -0,0 +1,28 @@
+import { AcademySignUpModal } from '../../academy/components/AcademySignUpModal/AcademySignUpModal';
+import { AcademySignUpModalProvider } from '../../academy/components/AcademySignUpModal/AcademySignUpModalProvider';
+import LocalElectionsHero from './LocalElectionsHero';
+import IsNowYourTime from './IsNowYourTime';
+import CampaignExperts from './CampaignExperts';
+import KickTheTires from './KickTheTires';
+import {
+  DEFAULT_SLANT_SECTION_COLORS,
+  SlantSection,
+} from '@shared/landing-pages/SlantSection';
+import { theme } from '../../../../tailwind.config';
```

Comment:
can you import that from its absolute path?

Reply:
https://github.com/thegoodparty/gp-webapp/pull/158/commits/0c40f1008de677ce26e1467424ae1f27c4cca7a7